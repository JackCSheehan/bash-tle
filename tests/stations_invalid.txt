AEROCUBE 12A            
1 43556U 18046C   22257.29976014  .00022692  00000+0  52641-3 0  9999
2 43556  51.6354 109.8054 0005697   3.0795 357.0225 15.42281973233012
AEROCUBE 12B            
1 43557U 18046D   22256.97717455  .00019746  00000+0  50152-3 0  9997
2 43557  51.6361 117.3198 0006047 358.2894   1.8069 15.39624388232835
Invalid line 1
MMSATS-1                
1 47976U 98067SJ  22257.77749356  .00035046  00000+0  38349-3 0  9996
2 47976  51.6386 235.1169 0002352  38.7093  23.9344 15.62947322 84214
Invalid line 2
