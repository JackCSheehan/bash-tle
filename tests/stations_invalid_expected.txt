AEROCUBE 12A,43556,U,18,046,C,22,257.29976014,.00022692,00000+0,52641-3,0,999,9,43556,51.6354,109.8054,0005697,3.0795,357.0225,15.42281973,23301,2
AEROCUBE 12B,43557,U,18,046,D,22,256.97717455,.00019746,00000+0,50152-3,0,999,7,43557,51.6361,117.3198,0006047,358.2894,1.8069,15.39624388,23283,5
MMSATS-1,47976,U,98,067,SJ,22,257.77749356,.00035046,00000+0,38349-3,0,999,6,47976,51.6386,235.1169,0002352,38.7093,23.9344,15.62947322,8421,4
